internal/encryption/config/config_merge.go

```diff
@@ -7,12 +7,18 @@ package config
 
 import (
 	"github.com/hashicorp/hcl/v2"
-	"github.com/opentofu/opentofu/internal/configs"
+	"github.com/opentofu/opentofu/internal/configs/hcl2shim"
```

Comment:
I'm a bit unhappy about having hcl2shim referenced in this way. I would prefer the encryption library to be standalone OR if not, then we should name hcl2shim something more... descriptive and open it up with this library.

Reply:
We discussed this in one of our syncs today. We'll be merging this PR and I will visit this renaming and documentation as a follow up PR 👍